src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -1272,6 +1282,7 @@ lVbRuntimePlus:
             Return New VisualBasicCommandLineArguments With
             {
                 .IsScriptRunner = IsScriptRunner,
+                .InteractiveMode = InteractiveMode Or (IsScriptRunner AndAlso sourceFiles.Count = 0),
```

Comment:
Consider adding a clarifying comment. For example, why presence of files is significant?